README.md

```diff
@@ -58,24 +58,21 @@ This will be the algorithm that will automatically generate a fireworks display
 
 # Fireworks Display Sequence Format
 
-Format will be in the following form
+Fireworks display sequence format will be in JSON. The general format is as follows:
 ```
-[Timestamp] TypeOfFirework1 xPosition Angle Speed, TypeOfFirework2 xPosition Angle Speed, etc...
-[Timestamp] ...
-etc...
+{display: [
```

Comment:
Why do we have a single top level key called `display`? Are there any other top level keys? If we don't have other top level keys then this object should just be an array to keep it simple.